### DISCUSSION TITLE
Overriding styles

### DISCUSSION THREAD

**Discussion by a19i23** [giorgosart/react-easy-edit#133] [Q&A]
I see the documentation for modifying styling but I'm not that great at CSS. Can you give some examples of how to do this, please? 

I don't use css files in react and typically override using MUI's`makeStyles()` or `withStyles()` Here is something I tried
```
const StyledEasyEdit = withStyles({
  root: () => {
    return {
      color: black,

      '& .easy-edit-inline-wrapper': {
         display: flex,
      },
    };
  },
})(EasyEdit);
```

Thanks for any help

**Comment by giorgosart** [comment]
Hey @a19i23 can you please provide me with a codepen or something? Not sure what MUI is, haven't used that myself before.

**Reply by a19i23** [reply]
Hey thanks for your response. I actually got it to work by overriding the css as documented by creating a css file and using the classnames described. I was trying to avoid that just to keep our code consistent but I just couldn't figure it out. Maybe it's not possible with this library. Thanks anyway
